pkgs/tools/misc/waylevel/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, fetchFromSourcehut, makeWrapper, rustPlatform, wayland}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "waylevel";
+  version = "1.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~shinyzenith";
+    repo = pname;
+    rev = version;
+    hash = "sha256-T2gqiRcKrKsvwGNnWrxR1Ga/VX4AyllYn1H25aIKt5s=";
+  };
+
+  cargoHash = "sha256-gw5m1/btJ5zZP04C7BCnHqEOUBoeu0whK8W7xA+xSQo=";
+
+  buildInputs = [ makeWrapper ];
+
+  postInstall = ''
+  wrapProgram $out/bin/waylevel \
+  --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ wayland ]}"
```

```suggestion
    wrapProgram $out/bin/waylevel \
    --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ wayland ]}"
```
can't we use patchelf here or patch the dlopen call?